scripts/ping_slack_about_package_release.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+if [[ ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+    curl -X POST -H "Content-type: application/json" \
+	 --data "{\"text\":\"💫 New *Rasa SDK* version ${TRAVIS_TAG} has been released! Changelog: https://github.com/RasaHQ/rasa-sdk/blob/${TRAVIS_TAG}/CHANGELOG.rst\"}" \
+	 "https://hooks.slack.com/services/T0GHWFTS8/BMTQQL47K/${SLACK_WEBHOOK_TOKEN}"
```

Comment:
The `SLACK_WEBHOOK_TOKEN ` needs to be added to travis envs

Reply:
I was wondering where those came from, I couldn't find them in either of the other repos, so thought it was just something in our travis setup somewhere. Where can i do that?

Reply:
added it to travis :+1: